Sprint1.py

```diff
@@ -1,10 +1,7 @@
-oceny = open("C:\\Users\\kdabek\\Desktop\\UpSkill - Python\\dane_wejsciowe\\sprint1\\grades1.txt", 'r').read()
+oceny = open("C:\\Users\\kdabek\\Desktop\\UpSkill - Python\\dane_wejsciowe\\sprint1\\grades1.txt", 'r')
 print(oceny)
-Cezary = ((3+5+1+3+1+1+3+4+6)/9)
-print(float(Cezary))
-Dionizy =((6+3+6+5.5+5+3.5+4)/7)
-print(float(Dionizy))
-Barnaba = ((5+4+5.5+4+6+3.5+1)/7)
-print(float(Barnaba))
-Albert = ((3+3+2+4+6+5+2)/7)
-print(float(Albert))
\ No newline at end of file
+content = oceny. read()
+print(content)
+content_list = content. split(",")

```

Comment:
should not contain space after `.`
```suggestion
content_list = content.split(",")
```

Reply:
This splits file by `,` imo you need also splitting by a newline first (`\n` character)